### DISCUSSION TITLE
Upgrade AWS SDK version in  conductor-awssqs-event-queue

### DISCUSSION THREAD

**Discussion by Set2005** [Netflix/conductor#3165] [Q&A] [answered]
Hi, current AWS SDK version doesn't support k8s IAM Service Account authentication/authorisation, I like to upgrade the SDK on my localhost, but not sure how. I've tried update the version (1.11.103), and run the Gradle tasks to rebuild the dependencies but it doesn't seem to work, the SDK still at 1.11.86, can I get some guidance plz?
Also created a Stackoverflow
https://stackoverflow.com/questions/73287035/how-to-update-gradle-project-dependencies-intellj?noredirect=1#comment129428342_73287035

```
ext {
    revActivation = '2.0.0'
    revAwaitility = '3.1.6'
    revAwsSdk = '1.11.103' (Updated)
```

**Comment by apanicker-nflx** [comment, marked as answer]
In order to update and use the new dependencies locally, please regenerate your locks using ```./gradlew generateLock saveLock```

**Reply by Set2005** [reply]
Thank you
